contracts/core/AuxiliaryBlockStore.sol

```diff
@@ -328,6 +368,29 @@ contract AuxiliaryBlockStore is BlockStore {
         valid_ = _transitionHash == expectedTransitionHash;
     }
 
+    /**
+     * @notice Marks a block in the block store as justified. The source and
+     *         the target are required to know when a block is finalised.
+     *         Only the polling place may call this method.
+     *
+     * @dev This is an external function which can be called by polling place.
+     *      This function call justify_ function which is internal defined in
+     *      super class
+     *
+     * @param _sourceBlockHash The block hash of the source of the super-
+     *                         majority link.
+     * @param _targetBlockHash The block hash of the block that is justified.
+     */
+    function justify(
+        bytes32 _sourceBlockHash,
+        bytes32 _targetBlockHash
+    )
+        external
+        onlyPollingPlace()
+    {
+        super.justify_(_sourceBlockHash, _targetBlockHash);
```

Comment:
Can `super.justify_()` revert or return an error? A comment addressing that would help here to understand that it is safe to call `updateKernel()` afterwards without any checks.

Reply:
Added the comments